src/ui/windows/TogglDesktop/TogglDesktop/ui/views/TimerEntryListView.xaml

```diff
@@ -4,20 +4,101 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:toggl="clr-namespace:TogglDesktop"
+             xmlns:vm="clr-namespace:TogglDesktop.ViewModels"
+             xmlns:mah="http://metro.mahapps.com/winfx/xaml/controls"
              mc:Ignorable="d" 
-             d:DesignHeight="300" d:DesignWidth="400">
+             d:DesignHeight="300" d:DesignWidth="400"
+             d:DataContext="{d:DesignInstance vm:TimerEntryListViewViewModel, IsDesignTimeCreatable=False}">

```

Comment:
I'm seeing newly introduced binding errors, might explain why switching the timeline flag to false doesn't do anything:
>System.Windows.Data Error: 40 : BindingExpression path error: 'IsTimelineViewEnabled' property not found on 'object' ''MainWindow' (Name='')'. BindingExpression:Path=IsTimelineViewEnabled; DataItem='MainWindow' (Name=''); target element is 'Border' (Name=''); target property is 'Visibility' (type 'Visibility')
System.Windows.Data Error: 40 : BindingExpression path error: 'SelectedTab' property not found on 'object' ''MainWindow' (Name='')'. BindingExpression:Path=SelectedTab; DataItem='MainWindow' (Name=''); target element is 'TabControl' (Name='TabControl'); target property is 'SelectedIndex' (type 'Int32')
System.Windows.Data Error: 40 : BindingExpression path error: 'IsTimelineViewEnabled' property not found on 'object' ''MainWindow' (Name='')'. BindingExpression:Path=IsTimelineViewEnabled; DataItem='MainWindow' (Name=''); target element is 'TabItem' (Name=''); target property is 'Visibility' (type 'Visibility')
System.Windows.Data Error: 40 : BindingExpression path error: 'IsTimelineViewEnabled' property not found on 'object' ''MainWindow' (Name='')'. BindingExpression:Path=IsTimelineViewEnabled; DataItem='MainWindow' (Name=''); target element is 'Border' (Name=''); target property is 'Visibility' (type 'Visibility')

Reply:
Ah, the binding errors are probably due to viewmodel not being assigned when the view is constructed.
I think in principle it's better to not delay the viewmodel construction. But I see that you've probably done this so that we don't call the library methods before the library is initialized. Not sure what's the best way to go about it, perhaps calling library methods should occur only in response to some events or observables.

Reply:
Yes, you're right. If I assign ViewModel in constructor, then a call to lib method may happen before context is initialized. I think this would be an endless source of bugs if we allow theoretically to call a lib method before full initialization, though I also don't like this solution.

Reply:
https://github.com/toggl-open-source/toggldesktop/commit/8bcab7c6ea26a36beaf8417ba3467e0b9c12f6ff
This is my proposal, initialize in the MainWindow's constructor, but after StartUI(). This way there are no binding errors.